modules/ROOT/pages/third-party-software-in-mule.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: third, party, software, mule, 3rd
 
-Mule runtime engine includes third-party software as part of the source code, examples, default services or as dependencies. Mule has dependencies on a subset of libraries from the included third-party software. Mule applications might also include additional third party dependencies managed through Maven, which are not included in the third-party software list shared below.
+Mule runtime engine includes third-party software as internal dependencies. Mule has dependencies on a subset of libraries from the included third-party software. Mule applications might also include additional third party dependencies managed through Maven, which are not included in the third-party software list shared below.
```

Comment:
I wonder if "as internal dependencies" (found a few times in this doc) should be reworded to "upon which it is internally dependent"?